### DISCUSSION TITLE
Some possible upgrades for the ComfyUI notebook

### DISCUSSION THREAD

**Discussion by derpmagician** [gutris1/segsmaker#32] [General]
ComfyUI_NestedNodeBuilder => https://github.com/ssitu/ComfyUI_NestedNodeBuilder is not neccesary since ComfyUI has something similar by itself
Theres is now a model for controlnet that works like swiss knife https://huggingface.co/xinsir/controlnet-union-sdxl-1.0 for all the cases

**Comment by gutris1** [comment]
for a simple custom nodes, please just do a git clone url 
![image](https://github.com/gutris1/segsmaker/assets/132797949/a83f8d33-e0bb-41f4-8fcf-ba7054f1d394)

for the controlnet, I'll add that to the widget

**Comment by gutris1** [comment]
there<br>
![image](https://github.com/gutris1/segsmaker/assets/132797949/e507cd63-1fbe-4a50-8c5a-f50bb4536ee2)


rerun this cell <br> then just run the controlnet.py <br><br>
![image](https://github.com/gutris1/segsmaker/assets/132797949/e49d7d10-5c8c-4475-babc-f067492617a9)
